corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -19,6 +19,14 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
             //set title of tab to application name
             if (menuResponse.breadcrumbs) {
                 document.title = menuResponse.breadcrumbs[0];
+
+                if (menuResponse.breadcrumbs.length === 2 && hqImport('hqwebapp/js/toggles').toggleEnabled('APP_ANALYTICS')) {
```

Comment:
This function gets called on every menu request, so it isn't always showing a case list. Can this code go somewhere more specific to case lists? Probably around [here](https://github.com/dimagi/commcare-hq/blob/8a73ab58201c6a5f06b41e51b09dd4a759990eb7/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/util.js#L119-L134) or inside the `CaseListView`?